services/catalog/tests/unit/with_dbs/conftest.py

```diff
@@ -109,7 +110,7 @@ async def product(
     """
     # NOTE: this fixture ignores products' group-id but it is fine for this test context
     assert product["group_id"] is None
-    async with insert_and_get_row_lifespan(
+    async with insert_and_get_row_lifespan(  # pylint:disable=contextmanager-generator-missing-cleanup
```

Comment:
thisone is an interesting one ...  here in the tests are probably not useful but it is something to take into account for the context managers https://pylint.pycqa.org/en/latest/user_guide/messages/warning/contextmanager-generator-missing-cleanup.html

i.e. we should not always happyly ignore it!